### PR TITLE
Clarify init questions about CA addresses

### PR DESCRIPTION
**Pull request by tashian**

### CODE REVIEW

File: command/ca/init.go [hunk 1]
```diff
@@ -523,9 +523,9 @@ func initAction(ctx *cli.Context) (err error) {
 	if pkiOnly {
 		pkiOpts = append(pkiOpts, pki.WithPKIOnly())
 	} else {
-		ui.Println("What DNS names or IP addresses would you like to add to your new CA?",
+		ui.Println("What DNS names and IP addresses will clients use to reach your CA?",
```
Comment (dopey) [inline command/ca/init.go:526]: This makes me think I need DNS names **and** IP addresses.

**Review by dopey — COMMENTED** [review]
As it's written, the new prompt makes me think I need both an IP address and a DNS name.

File: command/ca/init.go (hunk 1 above)
Comment (tashian) [reply command/ca/init.go]: good call, changed it back to `or`

**Comment by jdoss** [comment]
LGTM Carl. You are my hero.

**Review by jdoss — APPROVED** [review]

**Pull request merged**